### ISSUE TITLE
Ctrl-c from insert mode leaves cursor on same character (should move one left)

### ISSUE THREAD

**Issue by hacksoi** [VsVim/VsVim#2762]
Ctrl-c while in insert mode used to place the cursor one character to the left. It no longer does that. Note that using the Escape key from insert mode still places the cursor one character to the left. This only started happening recently (in the past month).

**Comment by jaredpar** [comment]
Thanks for reporting. Wonder if that is related to the abbrevations change.

**Comment by ricksladkey** [comment]
@jaredpar I would bet that the problem is related to the new "control C is the universal cancel" implementation.

**Comment by jaredpar** [comment]
True. That could definitely be it too. Had abbreviations on my brain when I was looking at this
